app/helpers/admin/users_helper.rb

```diff
@@ -45,5 +45,37 @@ def user_status(user)
         "Good Standing"
       end
     end
+
+    # Provides the remaining count when a limit for a resource is imposed on the UI.
+    #
+    #  @param [Integer] The total count
+    #  @param [Integer] A limit that we show
+    # @return [Integer] The overflow that is calculated
+    def overflow_count(count, imposed_limit: 0)
+      return if count <= imposed_limit
+
+      count - imposed_limit
+    end
+
+    # Returns a string for the organization tooltip outlining the first three organizations
+    # as a comma seperated values and then appending the remainding organizations (if any)
+    # as a count.
+    #
+    #  @param {Array} [array] The array of organization names
+    #  @param [Integer] The total count of organizations
+    #  @param [Integer] The limit of organizations that we show
+    # @return [String]
+    def organization_tooltip(organization_names, count, imposed_limit: 0)
+      str = organization_names.first(imposed_limit).join(", ").to_s
+
+      return str unless count > imposed_limit
+
+      overflow = overflow_count(count, imposed_limit: imposed_limit)
+      if overflow == 1
+        str + " & #{overflow_count(count, imposed_limit: imposed_limit)} other."
```

Comment:
It's a tiny thing but I don't think we want the full stop (it's not in the design, and I feel like it looks a bit weird when it's not a full sentence 🤔  I don't mind if this gets swept up in the later UI update though 

```suggestion
        str + " & #{overflow_count(count, imposed_limit: imposed_limit)} other"
```